public/video-ui/src/components/utils/VideoEmbed.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export function VideoEmbed({ sources }) {
+  return (
+    <video className="video-player" controls>
+      {sources.map(source => {
+        return (
+          <source key={source.src} src={source.src} type={source.mimeType} />
```

Comment:
Wtf Safari. Is there a way I can support it transparently, by putting `src` on the `video` tag or something?